### ISSUE TITLE
No USB after running Point Cloud Projection

### ISSUE THREAD

**Issue by iacisme** [luxonis/depthai-experiments#126]
Hi All;

When I run the **Point Cloud Projection** example, I get the folloiwng error:
![depthai_issues](https://user-images.githubusercontent.com/72644814/118077034-92785200-b370-11eb-891a-32ec202a5c2d.png)

Aftrwarrds, when I running any other demo, including ones in the **depthai-python** 'examples' folder I get this error:

![depthai_2](https://user-images.githubusercontent.com/72644814/118077127-c05d9680-b370-11eb-80ef-e5f77cf44ada.png)

After some trouble-shooting, what's going on is something in the OAK-D. If I unplugg and disconnect the device, and plug iback in, everything works again.

I tried **RGB-D and PCL** demo, and got this issue:

![errors](https://user-images.githubusercontent.com/72644814/118077715-e20b4d80-b371-11eb-8781-92b4af1e2672.png)

Long story short, it seems like running the examples in this repo have very unexpected results.

I've had to reboot my machine after running some these examples.

I know some others have had similar issues, but there isn't a clear ideas of how to address.

**Comment by Luxonis-Brandon** [comment]
Hi @iacisme ,

Sorry about the trouble here.  I -think- the root of the issue is that some of our examples (and in particular the `Point Cloud Projection` ([here](https://github.com/luxonis/depthai-experiments/tree/master/point-cloud-projection)) is using our Gen1 API.  And everything else is Gen2 (or most everything else).  And we've seen some really wonky behavior when switching back and forth on the same machine between Gen1 and Gen2.  Unplugging and restarting the DepthAI device between switching helps prevent thise.

But ultimately, we need to get everything to Gen2.  And I think we have a Gen2 point-cloud example (actually we have multiple), so we should probably deprecate this example and point to the Gen2 examples instead.

Thoughts on that @saching13 ?

Thanks - and sorry again about the trouble here,
-Brandon

**Comment by saching13** [comment]
point cloud examples are gen1 example and `03_depth_preview.py` is gen2. 
When changing between gen1 and gen2 you need to reinstall the depthai version of that and also unplug and connect the device.
`gen2-camera-demo` example [here](https://github.com/luxonis/depthai-experiments/tree/master/gen2-camera-demo#running-example-as-is) has point cloud visualization. 

And also sorry for the confusion and trouble.

**Comment by iacisme** [comment]
Hey guys, thanks for the responses. @saching13 I will try your link and get back to you ASAP. I really need something with point cloud visualizations so appreciate you sending that link.

To be clear, the following point cloud visualizations found in the **depthai-experiments** repository are gen1:
- pcl-projection-rgb
- point-cloud-projection

**gen2-camera-demo** did work, so will double check it again.

Thanks guys, this really helps understand more what was happening and why. I'll pay more attention to the name of the repo, I'm assuming you guys always prefix the gen2 stuff.

Therefore I'm going to assume now that if I don't see that prefix, it's gen1.

Cheers!
